app/screens/Terms.tsx

```diff
@@ -230,7 +216,7 @@ const Terms = ({ navigation }: Props): any => {
                       )}
                     </View>
                   </CheckboxItem>
-                  <LabelText>{t('tNccheckbox1')}</LabelText>
+                  <LabelText>{t('tNccheckbox2')} <CheckboxItemText onPress={goToTerms} style={styles.checkboxStyle}>{t('tNcTermsConditions')}</CheckboxItemText></LabelText>
```

Comment:
Is this change intentional? tNccheckbox1 to tNccheckbox2

Reply:
yes this is intentional as per new requirement.